### PR TITLE
Added WebAnimationsAPI-related browser globals

### LINKED ISSUE #87: New browser globals
The [Web Animations API spec](https://w3c.github.io/web-animations/) is being implemented in Firefox (parts have already shipped in FF46/47, and there's more to come) and Chrome (unsure about the version), and it's on Edge's roadmap.

This spec brings a number of new constructors available globally in a browser environment:

- KeyframeEffect
- KeyframeEffectReadOnly
- AnimationTimeline
- Animation
- AnimationEffectTiming
- AnimationEffectTimingReadOnly
- AnimationEffectReadOnly
- SharedKeyframeList

Maybe more that I have missed.
These should be added to the list of globals in this project. I'd be interested to contribute this if you let me know how to get started (seems like a simple enough change in the json file, but maybe I'm missing something).

It's my understanding that this is, in turn, used by eslint to validate script in browser environments.

### PR DESCRIPTION
**Pull request by captainbrosset**
Fixes #87 by introducing a list of new browser globals.
@birtles do you think this list is correct? I might have missed some things.

### CODE REVIEW

**Comment by birtles** [comment]
I think we're missing `DocumentTimeline` and `AnimationPlaybackEvent`.

**Comment by captainbrosset** [comment]
Thanks, I pushed a new squashed commit to this branch to add these 2 new globals.

File: globals.json [hunk 1]
```diff
@@ -215,6 +221,7 @@
 		"Crypto": false,
 		"CryptoKey": false,
 		"CSS": false,
+		"CSSAnimation": false,
```
Comment (nzakas) [inline globals.json]: Where do CSSAnimation and CSSTransition come from? They don't appear in the spec you referenced.

Comment (birtles) [inline globals.json]: https://drafts.csswg.org/css-animations-2/#cssanimation
https://drafts.csswg.org/css-transitions-2/#csstransition

Comment (nzakas) [inline globals.json]: @birtles thanks!

**Pull request merged**